src/matchbox/client/models/models.py

```diff
@@ -1,21 +1,42 @@
 """Functions and classes to define, run and register models."""
 
-from typing import Any, ParamSpec, TypeVar, overload
+import inspect
+from typing import ParamSpec, TypeVar, overload
 
 import polars as pl
 
 from matchbox.client import _handler
-from matchbox.client.models.dedupers.base import Deduper
-from matchbox.client.models.linkers.base import Linker
+from matchbox.client._settings import settings
+from matchbox.client.models import dedupers, linkers
+from matchbox.client.models.dedupers.base import Deduper, DeduperSettings
+from matchbox.client.models.linkers.base import Linker, LinkerSettings
+from matchbox.client.queries import Query
 from matchbox.client.results import Results
 from matchbox.common.dtos import ModelConfig, ModelType, Resolution
-from matchbox.common.exceptions import MatchboxResolutionNotFoundError
-from matchbox.common.graph import ModelResolutionName, ResolutionName, ResolutionType
+from matchbox.common.graph import ResolutionType
 from matchbox.common.logging import logger
 
 P = ParamSpec("P")
 R = TypeVar("R")
 
+default_dedupers = {
+    name: obj for name, obj in inspect.getmembers(dedupers, inspect.isclass)
+}
+default_linkers = {
+    name: obj for name, obj in inspect.getmembers(linkers, inspect.isclass)
+}
+
+
+_MODEL_CLASSES = {**default_dedupers, **default_linkers}
```

Comment:
Just roll it into one?

```suggestion
_MODEL_CLASSES = {
    **{name: obj for name, obj in inspect.getmembers(dedupers, inspect.isclass)},
    **{name: obj for name, obj in inspect.getmembers(linkers, inspect.isclass)}
}
```

Reply:
done